resources/odvd/ODVDFH16Truck.odvd

```diff
@@ -16,11 +16,12 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-message opendlv.proxy.reverefh16.ManualControl [id = 191] {
+message opendlv.proxy.reverefh16.ManualDriver [id = 191] {
```

Comment:
as support_for_dbc_v6 was merged in opendlv, we will need this PR merged as well to solve the naming changes